src/WebServer.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021, Kung-Yao Hsieh, Shang-Chen LIN, Yi-Hang Xie. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef HTTPLIB_SRC_WEBSERVER_H
+#define HTTPLIB_SRC_WEBSERVER_H
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#define DEFAULT_IP "127.0.0.1"
```

Comment:
Please leave a TODO to allow configuration of IP and port.

Reply:
Please make it a static constexpr member of your class.

Reply:
done